entities/src/main/java/com/flipkart/varadhi/entities/auth/UserContext.java

```diff
@@ -1,10 +1,24 @@
 package com.flipkart.varadhi.entities.auth;
 
+import java.util.Map;
+import java.util.Optional;
+
+/*
+    * UserContext represents the authenticated user. It is used to pass the user information across the system.
+    * This can be enhanced to include more details like context around the authentication.
+*/
+
 public interface UserContext {
 
     String getSubject();
 
     boolean isExpired();
 
-    // TODO: enhance it to include more details like context around the authentication
+    default Optional<String> getOrg() {
```

Comment:
what does it mean for the user to have Org?

Reply:
org here is associated with the tenants of AuthN. org/tenants have been used interchangeably by authN. 

for varadhi also, cross-org resource access should not be allowed.